scripts/tests/test_validate_docstrings.py

```diff
@@ -1052,6 +1052,13 @@ def test_raises_for_invalid_attribute_name(self, invalid_name):
         with pytest.raises(AttributeError, match=msg):
             validate_docstrings.Docstring(invalid_name)
 
+    @pytest.mark.parametrize('name', [
+        'pandas.Series.str.isdecimal', 'pandas.Series.str.islower'])
```

Comment:
it's a personal preference, but comparing
```python
@pytest.mark.parametrize('name', [
    'pandas.Series.str.isdecimal', 'pandas.Series.str.islower'])
```
with:
```python
@pytest.mark.parametrize('name', ['pandas.Series.str.isdecimal',
				  'pandas.Series.str.islower'])
```
I find the latter more clear.
```